app/views/shared/_navbar.html.erb

```diff
@@ -1,4 +1,4 @@
-<nav class="navbar navbar-expand-lg navbar-light bg-light mb-4">
+<nav class="navbar navbar-expand-lg navbar-light mb-3 sticky-top" style="background-color: #e3f2fd;">
```

Comment:
Especially with colors, I usually make a custom CSS class instead of an inline stlye b/c quite often those are used in more than one place.